lib/node/runner/checker_ballot_transaction.go

```diff
@@ -209,7 +213,9 @@ func ValidateTx(st *storage.LevelDBBackend, tx transaction.Transaction) (err err
 //   source = Account from where the transaction (and ops) come from
 //   tx = Transaction to check
 //
-func ValidateOp(st *storage.LevelDBBackend, source *block.BlockAccount, op transaction.Operation) (err error) {
+func ValidateOp(st *storage.LevelDBBackend, tx transaction.Transaction, op transaction.Operation) (err error) {
+	var source *block.BlockAccount
+	source, _ = block.GetBlockAccount(st, tx.B.Source)
```

Comment:
Missing error handling

Reply:
@Geod24 `ValidateOp` is called from `ValidateTx`.
In `ValidateTx` tx.B.Source is already checked before calling `ValidateOp`.
So I remove err check. Is it necessary to check tx.B.Source again?

Reply:
Oh wait, this function was previously taking the `BlockAccount` as input, so why moving it inside ?
Also, I don't think `tx` is necessary anymore, is it ?

Reply:
@Geod24 Yes you are right. I'll fix it.